### ISSUE TITLE
Not Working For Android Real Device, But Working on Emulator,

### ISSUE THREAD

**Issue by wajahat414** [enwi/dartzmq#17]
[dartzmq] Failed to load library zmq:  Invalid argument(s): Failed to load dynamic library 'libzmq.so': dlopen failed: cannot locate symbol "__aarch64_ldadd4_relax" referenced by "/data/app/~~MjLexgoFe5RqnQkIAbG4Cg==/com.example.trader-BdFk7vHBEbrjTXe0LKQ4MA==/lib/arm64/libzmq.so"...
[dartzmq] Failed to load library libzmq:  Invalid argument(s): Failed to load dynamic library 'liblibzmq.so': dlopen failed: library "liblibzmq.so" not found
[dartzmq] Failed to load library libzmq-v142-mt-4_3_5:  Invalid argument(s): Failed to load dynamic library 'liblibzmq-v142-mt-4_3_5.so': dlopen failed: library "liblibzmq-v142-mt-4_3_5.so" not found

**Comment by wajahat414** [comment]
@enwi  I  was reading about another issue in which it says zmq takes 1 second to  get Data,  does it mean that  in pub/sub when a publisher socket pushes  a data  subscriber socket will recieve after 1 second?

**Comment by wajahat414** [comment]
what if you try another approach just calling native platform zmq libraries  instead of using ffi  would'nt that be a better solution? as for every platform already zmq implementations are available, what do you think give me your opinion on that,
coz this way its not working

**Comment by enwi** [comment]
> @enwi I was reading about another issue in which it says zmq takes 1 second to get Data, does it mean that in pub/sub when a publisher socket pushes a data subscriber socket will recieve after 1 second?

In a worst case scenario it would take 1 second for the subscriber to receive the message

**Comment by enwi** [comment]
> what if you try another approach just calling native platform zmq libraries instead of using ffi would'nt that be a better solution? as for every platform already zmq implementations are available, what do you think give me your opinion on that, coz this way its not working

There is no dart implementation of the ZeroMQ protocol why I am using the shared c++ library, which should work fine

**Comment by wajahat414** [comment]
yes  I know, anyway, can you check why its is not working on real device,?
